src/protect-function-app/README.md

```diff
@@ -0,0 +1,126 @@
+<!-- TO PUBLISH TO DOCS SAMPLES BROWSER: UPDATE AND THEN UNCOMMENT THE YAML FRONT-MATTER SECTION BELOW -->
+<!--
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- Python
+page_type: sample
+name: "Python Azure Function that protects an HTTP trigger function  with Easy Auth and access token scope validation."
+description: "This Python Azure Function protects its own HTTP Trigger function with Easy Auth and access token scope validation. The code in this sample is used by one or more articles on docs.microsoft.com."
+products:
+- azure
+- azure-active-directory
+- azure-functions
+urlFragment: ms-identity-docs-code-functions-python
+---
+-->
+# Python | Azure Function | protect an API | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This Python Azure Function protects its own API (HTTP trigger) with the combination of the built-in authentication & authorization feature of Azure Functions (commonly known as "Easy Auth") and JWT access token scope validation.
+
+```console
+$ curl https://<your-function>.azurewebsites.net/api/greeting -H "Authorization: Bearer <valid-access-token>"
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+```
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
```

Comment:
nit: I think it is ok to mention that now users are going to need a subscription as well.

Reply:
Wouldn't saying that they need a deployed Azure Function cover that requirement already?  What value would adding a _pre-_ prereq bring to this list?

Reply:
The idea behind this is that I think it is common patter to introduce our users with something as shown below:

```bash
1. An Azure subscription. You can [open an account for free](https://azure.microsoft.com/free).
```

But yeah, asking for an Azure Function resource instance already deployed I think it is something that will enable our users to immediately infer this requirement.

Reply:
Another option could be to cross link to some another doc like the one for register apps while mentioning the need for an already-deployed Function App. 

This option looks more appealing to the README style we are following I guess.

Reply:
Just chatted with @ferantivero -- all good to close out this comment.  The already existing pre-req covers all suggestions above.